content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -109,7 +110,12 @@ the example application throughout this task.
                         request_host: request.host
     {{< /text >}}
 
-1. Apply the following annotation to all injected pods with the list of the
+
+    {{< text bash >}}
+    $ istioctl install -f custom_metrics.yaml
+    {{< /text >}}
+
+2. Apply the following annotation to all injected pods with the list of the
```

Comment:
Normally, we would leave this as a 1. and let the markdown handle the numbering. I believe there is an issue with tabs and numbering which requires the numbering to be done manually.